sdk/keyvault/Azure.Security.KeyVault.Secrets/tests/samples/Sample2_BackupAndRestore.cs

```diff
@@ -69,6 +65,18 @@ private void BackupAndRestoreSync(string keyVaultUrl)
             #endregion
 
             AssertSecretsEqual(storedSecret.Properties, restoreSecret);
+
+            // Delete and purge the restored secret.
+            operation = client.StartDeleteSecret(restoreSecret.Name);
```

Comment:
Same comment as in the async sample

Reply:
we should put this comment everywhere we are purging in a sample

Reply:
It's in the markdowns. I removed a bunch of the comments when we had to convert to the markdowns so we're not having to maintain duplication. I can, in this case, add this note, though.